usage/src/Volume/PET_CT_Overlay.jsx

```diff
@@ -349,6 +354,20 @@ function Example(props) {
             setValue={setColorPreset}
             style={{ top: '10px', left: '405px' }}
           />
+          <Slider
+            label='PET Level'
+            max={34611}
```

Comment:
Are these `max` constants based on the underlying data's range?

Reply:
Yes. Ideally, this should be set to max scalar range. Or some heuristic value based on it.